FileXTTest/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+cmake_minimum_required (VERSION 3.8)
+project ("FileXT Test")
+
+include_directories(${PROJECT_SOURCE_DIR}/../FileXT)
+file(GLOB SRC_FILES ${PROJECT_SOURCE_DIR}/*.cpp ${PROJECT_SOURCE_DIR}/*.h)
+
+add_executable(FileXTTest ${SRC_FILES})
+
+if (WIN32)
+    target_link_libraries(FileXTTest FileXT)
+endif (WIN32)
+if (UNIX)
+    target_link_libraries(FileXTTest FileXT dl)
```

Comment:
dl is needed specifically for the dladdr function. Funny enough we only need this on the executable project, not FileXT.